src/d/actor/d_a_obj_ebomzo.cpp

```diff
@@ -4,118 +4,299 @@
-}
+        /* 00000BBC-00000BDC      .text Mthd_Delete__Q211daObjEbomzo28@unnamed@d_a_obj_ebomzo_cpp@FPv  */
+        s32 Mthd_Delete(void* i_this) {
+            return static_cast<Act_c*>(i_this)->Mthd_Delete();
+        }
 
-/* 00000BDC-00000BFC       .text Mthd_Execute__Q211daObjEbomzo28@unnamed@d_a_obj_ebomzo_cpp@FPv */
-void Mthd_Execute(void*) {
-    /* Nonmatching */
-}
+        /* 00000BDC-00000BFC      .text Mthd_Execute__Q211daObjEbomzo28@unnamed@d_a_obj_ebomzo_cpp@FPv  */
+        s32 Mthd_Execute(void* i_this) {
+            return static_cast<Act_c*>(i_this)->MoveBGExecute();
+        }
 
-/* 00000BFC-00000C28       .text Mthd_Draw__Q211daObjEbomzo28@unnamed@d_a_obj_ebomzo_cpp@FPv */
-void Mthd_Draw(void*) {
-    /* Nonmatching */
-}
+        /* 00000BFC-00000C28      .text Mthd_Draw__Q211daObjEbomzo28@unnamed@d_a_obj_ebomzo_cpp@FPv  */
+        s32 Mthd_Draw(void* i_this) {
+            return static_cast<Act_c*>(i_this)->Draw();
+        }
 
-/* 00000C28-00000C54       .text Mthd_IsDelete__Q211daObjEbomzo28@unnamed@d_a_obj_ebomzo_cpp@FPv */
-void Mthd_IsDelete(void*) {
-    /* Nonmatching */
-}
+        /* 00000C28-00000C54      .text Mthd_IsDelete__Q211daObjEbomzo28@unnamed@d_a_obj_ebomzo_cpp@FPv  */
+        s32 Mthd_IsDelete(void* i_this) {
+            return static_cast<Act_c*>(i_this)->IsDelete();
+        }
 
-static actor_method_class Mthd_Ebomzo = {
-    (process_method_func)Mthd_Create,
-    (process_method_func)Mthd_Delete,
-    (process_method_func)Mthd_Execute,
-    (process_method_func)Mthd_IsDelete,
-    (process_method_func)Mthd_Draw,
-};
-}; // namespace
-}; // namespace daObjEbomzo
+        static actor_method_class Mthd_Ebomzo = {
+            (process_method_func)Mthd_Create,
+            (process_method_func)Mthd_Delete,
+            (process_method_func)Mthd_Execute,
+            (process_method_func)Mthd_IsDelete,
+            (process_method_func)Mthd_Draw,
+        };
+    }
+}
 
 actor_process_profile_definition g_profile_Obj_Ebomzo = {
     /* LayerID      */ fpcLy_CURRENT_e,
-    /* ListID       */ 0x0003,
-    /* ListPrio     */ fpcPi_CURRENT_e,
+    /* ListID       */ 3,
+    /* ListPrio     */ fpcLy_CURRENT_e,
     /* ProcName     */ PROC_Obj_Ebomzo,
     /* Proc SubMtd  */ &g_fpcLf_Method.base,
-    /* Size         */ sizeof(daObjEbomzo::Act_c),
+    /* Size         */ sizeof(daObjEbomzo::Act_c), //Should be 0x448
     /* SizeOther    */ 0,
     /* Parameters   */ 0,
     /* Leaf SubMtd  */ &g_fopAc_Method.base,
     /* Priority     */ 0x007C,
-    /* Actor SubMtd */ &daObjEbomzo::Mthd_Ebomzo,
-    /* Status       */ fopAcStts_NOCULLEXEC_e | fopAcStts_CULL_e | fopAcStts_UNK40000_e,
+    /* Actor SubMtd */ &daObjEbomzo::Mthd_Ebomzo, //Mthd_Ebomzo__Q211daObjEbomzo28@unnamed@d_a_obj_ebomzo_cpp@,
+    /* Status       */ fopAcStts_NOCULLEXEC_e | fopAcStts_CULL_e | fopAcStts_UNK40000_e, //0x00040180
     /* Group        */ fopAc_ACTOR_e,
     /* CullType     */ fopAc_CULLBOX_CUSTOM_e,
 };
```

Comment:
can you revert the changes to the actor profile so the style stays consistent across all actors?